settings.gradle

```diff
@@ -33,6 +33,13 @@ develocity {
             // Alternatively, the build scan will provide the hostname for troubleshooting host-specific issues.
             ipAddresses { addresses -> addresses.collect { address -> "0.0.0.0"} }
         }
+        if (System.getenv("JENKINS_URL")) {
```

Comment:
Maybe we can reuse the existent def? see https://github.com/apache/kafka/blob/trunk/settings.gradle#L21